Shared/Realm.Sync.Shared/Handles/SessionHandle.cs

```diff
@@ -31,20 +32,90 @@ private static class NativeMethods
                                                            [MarshalAs(UnmanagedType.LPWStr)] string server_path, IntPtr server_path_len,
                                                            out NativeException ex);
 
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_syncsession_get_user", CallingConvention = CallingConvention.Cdecl)]
+            public static extern IntPtr get_user(SessionHandle session);
+
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_syncsession_get_uri", CallingConvention = CallingConvention.Cdecl)]
+            public static extern IntPtr get_uri(SessionHandle session, IntPtr buffer, IntPtr buffer_length, out NativeException ex);
+
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_syncsession_get_state", CallingConvention = CallingConvention.Cdecl)]
+            public static extern SessionState get_state(SessionHandle session, out NativeException ex);
+
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_syncsession_get_from_realm", CallingConvention = CallingConvention.Cdecl)]
+            public static extern IntPtr get_from_realm(SharedRealmHandle realm, out NativeException ex);
+
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_syncsession_get_raw_pointer", CallingConvention = CallingConvention.Cdecl)]
+            public static extern IntPtr get_raw_pointer(SessionHandle session);
+
             [DllImport(InteropConfig.DLL_NAME, EntryPoint = "realm_syncsession_destroy", CallingConvention = CallingConvention.Cdecl)]
             public static extern void destroy(IntPtr handle);
         }
 
+        public SyncUserHandle User
```

Comment:
Should those be properties? This is internal API, so it feels they'd be more suited as methods.